modules/security-hosts-vms-rhcos.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-hosts-vms.adoc
+
+[id="security-hosts-vms-rhcos_{context}"]
+= How containers are secured on {op-system-first}
+
+Containers simplify the act of deploying many applications to run on the same host,
+using the same kernel and container runtime to spin up each container.
+The applications can be owned by many users and, because they are kept
+separate, can run different, and even incompatible, versions of those applications
+at the same time without issue.
+
+In Linux, containers are just a special type of process, so securing
+containers is similar in many ways to securing any other running process.
+An environment for running containers starts with an operating system (OS)
+that can secure the host kernel from
+containers and other processes running on the host, as well as
+secure containers from each other.
+
+Because {product-title} {product-version} runs on {op-system} hosts,
+with the option of using Red Hat Enterprise Linux (RHEL) as worker nodes,
+the following concepts apply by default to any deployed {product-title}
+cluster. These features are at the core of what make containers secure on the platform.
```

Comment:
I realize that this is the container security guide :) but I know that we often need to be careful and when we say that things are "secure". Suggest just going through the PR to make sure that we're careful of the usage, and changing, if necessary. I don't have a good example wording to suggest right now, but I can think on it.

Reply:
I changed it to this: "These RHEL security features are at the core of what
makes running containers in OpenShift more secure:"